package.json

```diff
@@ -627,17 +618,17 @@
             },
             "default": {}
           },
-          "dataEditor.omegaEditPort": {
+          "dataEditorOmegaEditPort": {
```

Comment:
I'm unsure what the schema definitions in this `configuration` section are for. They aren't official sections listed under https://code.visualstudio.com/api/extension-guides/debugger-extension#anatomy-of-the-package.json-of-a-debugger-extension. 

Looking around, it seems to be a custom configuration section we use to provide default values(?). If so, it can probably be removed and the `contributes/debuggers/configurationAttributes` section can be the sole location of launch schema and defaults.

This should probably be a different ticket for a different release.

Reply:
Yeah this what it is for. However, it was separated from `configurationAttributes` as we were using it for defaults and making sure the type is right initially. I don't think we want to move them into `configurationAttributes`, doing this allows the user to change their values, but they are only there for initial values of object attributes. I think we actually wouldn't want the user to have the ability to set these item, at least some of them since it really wouldn't do anything.